HPXMLtoOpenStudio/resources/BaseElements.xsd

```diff
@@ -5409,6 +5412,8 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:element name="HVACSystemsServed" type="LocalReference" minOccurs="0" maxOccurs="unbounded"/>
+			<xs:element maxOccurs="1" minOccurs="0" name="HeatingSeason" type="Season"/>
+			<xs:element minOccurs="0" name="CoolingSeason" type="Season"/>
```

Comment:
Inconsistency here. Make both of them: `maxOccurs="1" minOccurs="0"`

Reply:
I saw this too. I double-checked values from the oxygen ui. Not sure why it recorded them in the xsd like that...